packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -774,6 +774,7 @@ export class ContainerRuntime
 	 * - containerRuntimeCtor - Constructor to use to create the ContainerRuntime instance.
 	 * This allows mixin classes to leverage this method to define their own async initializer.
 	 * - provideEntryPoint - Promise that resolves to an object which will act as entryPoint for the Container.
+	 * - minVersionForCollab - Minimum version of the FF runtime that is required to collaborate on new documents.
```

Comment:
Why do you say "to collaborate on new documents"? I expected it to say "to collaborate with this client"

Reply:
I was actually just copying from an existing comment but this is an interesting point. Practically speaking, collaborating with new documents is a good way to describe it from the customer's POV (IMO), but for our internal comments its not exactly accurate. Updated in latest commit